src/components/search-section/search-section.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Container from '../container/container';
+import styles from './styles.module.scss';
+
+import SearchIcon from '../../assets/search-icon.png';
```

Comment:
susikonfiguruokite global paths, ir pakeiskiti VISAS vietas projekte kur yra naudojami salyginiai keliai pasiekti GLOBALIEM konstruktam;
https://stackoverflow.com/questions/68217795/vite-resolve-alias-how-to-resolve-paths